docs/reference2/sql-syntax/data-types/numeric.md

```diff
@@ -30,6 +28,18 @@ The `double` data type is an inexact, variable-precision numeric type. The term
 "inexact" means an approximate value is stored. Storing values outside of its
 bounds of capacity will result in an error.
 
+## Valid ranges
+
+Numeric data types have the same valid minimum and maximum values as their
+corresponding Java types. The following table shows valid ranges for the
+integer and floating-point types. 
+
+| type     | minimum value | maximum value
```

Comment:
Can we move these to the top table to keep them in one place?

Reply:
I moved this into the main table and tried different arrangements, but still it renders a bit awkwardly: 
![image](https://user-images.githubusercontent.com/12521043/89457481-50075200-d71a-11ea-97d0-604faf5f8207.png).
Columns get a bit cramped and line breaks happen in weird places. Probably okay for now, but we might want to revisit.

Reply:
Np, we can figure out column widths before final merge.